src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -3468,7 +3468,8 @@ namespace FanCoilUnits {
                           int const ControlledZoneNum,   // ZoneEquipConfig index
                           bool const FirstHVACIteration, // flag for 1st HVAV iteration in the time step
                           Real64 &LoadMet,               // load met by unit (watts)
-                          Optional<Real64> PLR           // Part Load Ratio, fraction of time step fancoil is on
+                          Optional<Real64> PLR,          // Part Load Ratio, fraction of time step fancoil is on
+                          Real64 eHeatCoilCyclingR       // electric heating coil cycling ratio  used with MultiSpeedFan capacity control 
```

Comment:
Added new argument *eHeatCoilCyclingR* for electric heating coil cycling ratio. This variable is used to cycle On/Off the electric heating coil when the *Fancoilunit* is running at the lowest speed and the *Fan* operating mode is ontinuous fan and cycling coil.